src/main/java/com/banking/bankservice/mapper/impl/AccountMapperImpl.java

```diff
@@ -14,7 +14,6 @@ public Account fromDto(AccountRequestDto accountRequestDto) {
         Account account = new Account();
         account.setAccountNumber(accountRequestDto.getAccountNumber());
         account.setBalance(accountRequestDto.getBalance());
-        account.setActive(true);
         account.setCurrencyType(Currency.valueOf(accountRequestDto.getCurrencyType().toString()));
```

Comment:
account.setCurrencyType(accountRequestDto.getCurrencyType());

Reply:
Okay, sorry. It was stupid question